pkg/cli/cmd/app/delete/delete_test.go

```diff
@@ -44,7 +45,12 @@ func Test_Validate(t *testing.T) {
 			ExpectedValid: true,
 			ConfigHolder: framework.ConfigHolder{
 				ConfigFilePath: "",
-				Config:         radcli.LoadConfigWithWorkspaceAndApplication(t),
+				Config:         radcli.LoadConfigWithWorkspace(t),
+				DirectoryConfig: &config.DirectoryConfig{
+					Workspace: config.DirectoryWorkspaceConfig{
+						Application: "test-application",
+					},
+				},
```

Comment:
This is configuring the test is basically the same way it was configured prior to the approval. 

We have both per-user and per-directory configuration now, so the per-user configuration for the app was redundant.